d2/src/main/java/com/linkedin/d2/balancer/strategies/LoadBalancerStrategy.java

```diff
@@ -58,7 +59,7 @@ TrackerClient getTrackerClient(Request request,
                                  RequestContext requestContext,
                                  long clusterGenerationId,
                                  int partitionId,
-                                 List<TrackerClient> trackerClients);
+                                 Map<URI, TrackerClient> trackerClients);
```

Comment:
Will this change in public method bring any backward-incompatible issue for clients? Is there any direct usage of this method from other mps?

Reply:
It's usage is internal to D2 so should be good, checked on go/code.